P/pprof/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+
+name = "pprof"
+
+# PProf doesn't have proper versions yet
+# so we use Go's pseudo version
+# `TZ=UTC git show --quiet --date='format-local:%Y%m%d%H%M%S' --format="%cd" $hash`
+
+hash = "f9b734f9ee64d0f5b63636a45cc77ed2744997ab"
+timestamp = "20191205061153"
+version = v"0.0.0-$timestamp-$hash"
```

Comment:
I usually do `v2019.12.05` for things like this.  We usually don't care about granularity smaller than a day anyway.

Reply:
There will be hopefully a version v`1.0.0`, since Go recently started to properly version things.
How about we drop the hash, but otherwise keep the pseudo-version format?